docs/source/essentials/normalized-cache.mdx

```diff
@@ -0,0 +1,343 @@
+---
+title: Normalized cache
+---
+
+import {MultiCodeBlock} from 'gatsby-theme-apollo-docs';
+
+Apollo Android has 2 different kinds of caches: HTTP cache and normalized cache. The HTTP cache is easier to setup but also has more limitations. This page focuses on the normalized cache. If you're looking for a simpler albeit coarser cache, take a look at the [HTTP cache](http-cache).
+
+## Data Normalization:
+
+The normalized cache stores objects by ID.
+
+```graphl
+query BookWithAuthorName {
+  favoriteBook {
+    id
+    title
+    author {
+      id
+      name
+    }
+  }
+}
+
+query AuthorById($id: String!) {
+  author(id: $id) {
+      id
+      name
+    }
+  }
+}
+```
+
+In the above example, requesting the author of your favorite book with the `AuthorById` query will return a result from the cache if you requested your favorite book before. This works because the author is stored only once in the cache and all the fields where retrieved in the initial BookWithAuthorName query. If you were to request more fields, like `birthdate` for an example, that wouldn't work anymore.BookWithAuthorName
```

Comment:
`.BookWithAuthorName` should be deleted?

Reply:
Urg, thanks